department-of-veterans-affairs/caseflow
# ClayS/APPEALS-23398

Resolves [APPEALS-23398](https://jira.devops.va.gov/browse/APPEALS-23398)

# Description
Changes the text displayed when a claimant does not have a POA on the add issues page.

## Acceptance Criteria
- [ ] During the intake of a Higher-Level Review (VA Form 20-0996) or Supplemental Claim (VA Form 20-0995) with a VHA Benefit Type, the Appellant’s Power of Attorney is listed in Step 4, Add Issues.
    - [ ] Updates to Claimant’s POA row:
        - [ ] If claimant has an associated POA in VBMS, the POA/Attorney Name will be displayed:
            - [ ] <Attorney>
        - [ ] If claimant has no associated POA in VBMS, the following text will be displayed:
            - [ ] Informational text: No known POA 


## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Create a veteran with no POA: `FactoryBot.create(:veteran, participant_id: "NO_POA111111113", file_number: "111111113")
`
3. search for veteran by file number (111111113) in intake
4. Choose vha benefit type
5. On the add issues page, verify that the 'No known POA' text displays
6. Cancel intake and repeat with a different benefit type to make sure the original text('Intake does not have a Form 21-22') displays 



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1106" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/a86e4e04-c8c8-45eb-b7c4-0d7af7072b7b">
|
<img width="1094" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/6730236f-482d-4417-be6d-fe16541b0d24">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
